oblt-cli/cluster-credentials/action.yml

```diff
@@ -0,0 +1,41 @@
+name: 'oblt-cli/cluster-credentials'
+description: 'Run the oblt-cli wrapper to retrieve the credentials to connect to the given cluster'
+inputs:
+  cluster-name:
+    description: 'The cluster name'
+    required: false
+  cluster-info-file:
+    description: 'The cluster info file (absolute path)'
+    required: false
+  github-token:
+    description: 'The GitHub access token.'
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - uses: elastic/oblt-actions/oblt-cli/cluster-name-validation@v1.2.0
+      id: validation
+      with:
+        cluster-name: ${{ inputs.cluster-name }}
+        cluster-info-file: ${{ inputs.cluster-info-file }}
+
+    - uses: elastic/oblt-actions/oblt-cli/setup@v1.1.0
```

Comment:
ditto. this one uses 1.1.0 and the previous one 1.2.0

Reply:
I chose this because the `oblt-cli/setup` was released in v1.1.0